src/functions.php

```diff
@@ -736,3 +736,30 @@ function parse($value)
 {
     return Context::get()->getConfig()->parse($value);
 }
+
+function locateBinaryPath($name)
+{
+    $nameEscaped = escapeshellarg($name);
+
+    // Try `command`, should cover all Bourne-like shells
+    if (commandExist("command")) {
+        return run("command -v $nameEscaped")->toString();
+    }
+
+    // Try `which`, should cover most other cases
+    if (commandExist("which")) {
```

Comment:
For the sake of absolute safety just tested this on Debian Wheezy, Fedora24/25, Ubuntu Xenial/Trusty, Centos7 and latest Alpine in docker, and even Windows Subsystem for Linux (don't ask). command -v is supported by all. Centos 7, F24, F25 does not have which installed by default. Debian derivs seem all to have which, but also command, and command being builtin is probably the saner choice for default.